packages/forma-36-website/src/pages/guidelines/copy/date-and-time/index.mdx

```diff
@@ -0,0 +1,239 @@
+    </TableRow>
+    <TableRow>
+      <TableCell>Weekday</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        Mon, 12 Aug
+        <br />
+        Wed, 14 Aug at 11:00 pm
+      </TableCell>
+      <TableCell>
+        Weekdays should be used when users need to compare events that happen in
+        a daily context, always next to absolute date and time and separated
+        with a comma.
+      </TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## Ranges
+
+To show a date or time range, use an **en dash** (Unicode: U+2013, html: `&ndash;`) with spaces around it.
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Examples</TableCell>
+      <TableCell style={{ width: '50%' }}></TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>8:00 am – 2:00 pm</TableCell>
+      <TableCell></TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Date</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        30 Mar – 30 Apr
+        <br />
+        8 Jul – 13 Aug 2019
+        <br />
+        13 Dec 2018 – 8 Aug 2019
+      </TableCell>
+      <TableCell>Omit the year if the range is in the same year.</TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## Don't use
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Example</TableCell>
+      <TableCell style={{ width: '50%' }}></TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Relative day</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        Last Tuesday
+        <br />
+        Today
+      </TableCell>
+      <TableCell>
+        Not expressed as relative to <i>now</i>.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute date</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        10/4/2019
+        <br />
+        10.4.2019.
+      </TableCell>
+      <TableCell>Not clear which number is day which is month.</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute time or date</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        08:00
+        <br />
+        02 Jan 2019
+      </TableCell>
+      <TableCell>
+        Don't use the leading zero in single-digit hours or days.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute date</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>4th Jan 2019</TableCell>
+      <TableCell>
+        Don't add ordinal number suffixes such as -st, -nd, -rd or -th.
+      </TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## Moment.js formats
```

Comment:
I think we can remove this section entirely as it's an implementation detail of a specific third-party package. What we could do in the future though is create some time/date components and link those to these guidelines. I don't think this work would be dependent on getting this PR merged though.